src/components/dashboard/FeedItems/EventStyles.js

```diff
@@ -1,22 +1,46 @@
 import { StyleSheet } from 'react-native'
 import normalize from 'react-native-elements/src/helpers/normalizeText'
+
 export const listStyles = StyleSheet.create({
   row: {
     flexDirection: 'row',
-    marginBottom: 5,
-    backgroundColor: 'rgb(238, 238, 239)',
+    marginBottom: normalize(4),
+    backgroundColor: '#fff',
     borderRadius: normalize(8),
-    overflow: 'hidden'
+    overflow: 'hidden',
+    shadowColor: '#000',
+    shadowOffset: { width: 0, height: normalize(2) },
+    shadowOpacity: 0.16,
+    shadowRadius: normalize(4),
+    elevation: 1,
+    height: normalize(84),
+    maxHeight: normalize(84)
   },
   rowContent: {
-    padding: normalize(5),
     justifyContent: 'center',
     alignItems: 'center',
     flex: 1,
-    borderLeftWidth: normalize(8),
-    borderLeftColor: 'rgb(186, 186, 186)'
+    paddingLeft: normalize(8),
```

Comment:
This paddings should be comming from the theme I think. I've created a defaultMargin property there. I will make a PR for using theme props on style creation.

Reply:
Done